src/ontology/foodon-edit.owl

```diff
@@ -18198,6 +18198,142 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://en.wikipedia.org/wiki/Veal</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-07-08T20:43:02Z</dc:date>
+        <rdfs:label xml:lang="en">bob veal</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003911 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003911">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003083"/>
+        <obo:IAO_0000115 xml:lang="en">A meat of a calf that is raised on a fortified milk formula diet plus solid feed.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://en.wikipedia.org/wiki/Veal</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-07T20:53:21Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">formula-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">milk fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">milk-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">special fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">special-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">white veal</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">formula fed veal</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003912 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003912">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03450006"/>
+        <obo:IAO_0000115 xml:lang="en">A Japanese cooking technique in which food is broiled or grilled with a glaze of soy sauce, mirin, and sugar.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://en.wikipedia.org/wiki/Teriyaki</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-07T21:03:32Z</dc:date>
+        <rdfs:label xml:lang="en">teriyaki</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003913 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003913">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03309952"/>
+        <obo:IAO_0000115 xml:lang="en">An ingredient or a combination of ingredients containing extra minerals, vitamins, and other compounds, is added to the cattle feed in order to meet specific needs.</obo:IAO_0000115>
```

Comment:
How about "An animal feed supplement for cattle." And lets have an "animal feed supplement" class - the higher class - with definition something like: "A feed supplement for domesticated animals which contains minerals, vitamins, and other compounds intended to provide nutrition or other benefits."

Reply:
Okay, I think then the new class 'animal feed supplement' should be a subclass of higher class 'food product for animal'. And will have a child 'animal feed supplement for cattle'. 
Does it make sense to also add 'cattle feed' as a subclass for 'animal feed supplement for cattle' or would it be ambigious?

Reply:
The supplements are things added to the feed.  So cattle feed should not be a subclass of any supplement.

Reply:
Sorry I just realized that I wrote it wrong, I meant 'cattle feed' as an additional parent class of 'animal feed supplement for cattle' along with 'animal feed supplement' as a parent class? As you also just pointed it out that supplements are added to the feed.